tests/conftest.py

```
@@ -171,22 +183,16 @@ def download_models(model_list=None, ignore_weights=False):
-                    # Download the full model snapshot (includes all files)
-                    snapshot_download(
-                        repo_id=model_id,
-                        token=hf_token,
-                    )
-                logging.info(f"Successfully pre-downloaded: {model_id}")
+                    snapshot_download(**dl_kwargs)
+                logging.info(f"Successfully pre-downloaded: {spec.repo_id}{rev_info}")
 
             except Exception as e:
-                logging.error(f"Failed to pre-download {model_id}: {e}")
+                logging.error(f"Failed to pre-download {spec.repo_id}: {e}")
```

Ruff (0.15.2)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)